src/resources/js/flexible-tickets/modules/filters.js

```diff
@@ -216,3 +216,28 @@ wp.hooks.addFilter (
     'tec.tickets.flexibleTickets',
     filterTicketsControlsMappedProps
 )
+
+/**
+ *
+ * Forces the Prompt Message component to show on Recurring Events if they are part of a Series.
+ *
+ * @since TBD
+ *
+ * @param {ReactNode} component
+ * @returns
+ */
+function filterDisplaySeriesPassesPromptMessage( component ) {
+	const hasSeriesPasses = ( TECFtEditorData?.series?.seriesPassesCount || 0 ) > 0;
+
+	if ( hasSeriesPasses ) {
+		return component;
```

Comment:
If there are Seris Passes return the mapped properties unaltered?

Reply:
Thanks @lucatume , I applied your suggestion in `filterTicketsContainerMappedProps`